test/multiverse/lib/multiverse/envfile.rb

```diff
@@ -133,6 +133,34 @@ def serialize?
       @serialize
     end
 
+    # add Rails Edge to the array of gem versions for testing,
+    # unless we're operating in a PR workflow context
+    def prepend_rails_edge(gem_version_array = [])
```

Comment:
I'm so glad to see the earlier work on Envfiles could make adding this method a bit easier! I like the implementation.

Even though this is an accurate use of prepend in the English language, I feel a bit uncomfortable with this method name. In Ruby contexts, I equate "prepend" with the "prepend" method, and I don't see that method being called here.

Cueing off the comments surrounding this method, perhaps `unshift_rails_edge` or `add_rails_edge` could be a better fit?

Reply:
Heh. I wanted to avoid confusing non-Rubyists with the naming but used a name that would instead confuse Rubyists. Went with `unshift_rails_edge` in [a63e5cc](https://github.com/newrelic/newrelic-ruby-agent/pull/2248/commits/a63e5ccc5542bc9ab3e31b2a026c1c865317fef5)